src/main/js/store/reducers/system.js

```diff
@@ -20,6 +40,50 @@ const config = (state = initialState, action) => {
             return Object.assign({}, state, {
                 fetching: false
             });
+        case SYSTEM_SETUP_FETCHING:
+            return Object.assign({}, state, {
+                fetchingSetup: true,
+                updateStatus: 'FETCHING'
+            });
+        case SYSTEM_SETUP_FETCH_REDIRECTED:
+            return Object.assign({}, state, {
+                fetchingSetup: false,
+                setupRedirect: true
```

Comment:
In all cases that don't set updateStatus to a value we should explicitly set it to an empty string. Otherwise it will carry over in the state and that has caused issues for us